### DISCUSSION TITLE
Using in SolidJS?

### DISCUSSION THREAD

**Discussion by tylerkneidl** [diegomura/react-pdf#2294] [General]
How hard would it be to use this in Solid JS? I'm looking to move to Solid but I love react-pdf and would hate to reinvent the wheel!  Could an adapter be made/used?  Is it impossible to use without a full rewrite?

I don't know enough about solid to say, so hoping maybe someone here knows a little (lot) more than I do.

**Comment by tonypee** [comment]
interested too
